build.gradle

```diff
@@ -189,7 +182,11 @@ shadowJar {
 		//exclude(dependency('org.slf4j:slf4j-api:.*'))
 		exclude(dependency('org.slf4j:slf4j-nop:.*'))
 		exclude(dependency('org.jboss.logging:jboss-logging:.*'))
+		exclude(dependency('eu.maveniverse.maven.mima:.*'))
+		exclude(dependency('eu.maveniverse.maven.mima.runtime:.*'))
+		exclude(dependency('org.apache.maven.resolver:.*'))
```

Comment:
These lines now with static-uber runtime could be removed, as minimize removed almost all of resolver (due implicit uses via JSR330 that minimize was unaware of).

Reply:
Hm, given that mima.runtime goes in via ServiceLocator, maybe not.... (unsure does minimize plugin "knows" about ServiceLocator pattern, as it does not know about Sisu index either).